### DISCUSSION TITLE
Multiprotocol

### DISCUSSION THREAD

**Discussion by Timofffee** [Minestom/Minestom#1433] [Help] [answered]
we are making an RPG server that will soon be ready for the first launch. however, it is made on Paper. already now I understand that this is a disgusting idea and therefore I would like to gradually transfer the entire server to minestom. however, rewriting the entire server now is a very bad idea. therefore, I decided to make sure that the main world remains as it is now, and some additional locations will be implemented through minestom. 

but there is a problem. for some reason, the main server was created on 1.17.1, and minestorm is built on the latest version of the kernel. What should I do? use a plugin in bungeecord/velocity/waterfall that allows you to use multiple versions? or is there some way to solve this by minestom itself (to lower the protocol)?

**Comment by GoldenCookieGIT** [comment, marked as answer]
You can use via version on a proxy and it'll work just fine
